src/mbed_os_tools/detect/platform_database.py

```diff
@@ -177,6 +177,8 @@
         u"0855": u"DISCO_L562QE",
         u"0879": u"NUCLEO_F756ZG",
         u"0885": u"B_L4S5I_IOT01A",
+        u"0886": u"NUCLEO_U575ZI_Q",
+        u"0887": u"DISCO_U585I_IOT02A",
```

Comment:
Note this doesn't match with what we have in the internal database:
`DISCO_B_U585I_IOT02A`
What do you prefer us to use?

Reply:
I changed to B_U585I_IOT02A :-)

Could you update your database like this ?

Thx

Reply:
Done